docs/guides/admin/docs/workflowoperationhandlers/cut-marks-to-smil-woh.md

```diff
@@ -0,0 +1,44 @@
+CutMarksToSmilWorkflowOperationHandler
```

Comment:
This is missing from the [list of workflow operation handlers](https://docs.opencast.org/develop/admin/#workflowoperationhandlers/#default-workflow-operations)